### PR TITLE
CNV-20526: RN CNV integration with OpenShift Pipelines

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.12

Issue: [CNV-20526](https://issues.redhat.com//browse/CNV-20526)

Link to docs preview: https://54302--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-12-release-notes.html#virt-4-12-technology-preview

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: virt/virt-4-12-release-notes.adoc [hunk 1]
```diff
@@ -179,6 +179,13 @@ Some features in this release are currently in Technology Preview. These experim
 
 link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
 
+//CNV-20526
+* The Tekton Tasks Operator (TTO) now xref:../virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc#virt-managing-vms-openshift-pipelines[integrates {VirtProductName} with {pipelines-title}]. TTO includes cluster tasks and example pipelines that allow you to:
++
+** Create and manage virtual machines (VMs), persistent volume claims (PVCs), and data volumes
+** Run commands in VMs
+** Manipulate disk images with `libguestfs` tools
```
Comment (ksimon1) [inline virt/virt-4-12-release-notes.adoc:187]: would be please possible to mention here, that with example pipelines user can try install windows 10 vm from iso file?

Comment (geetikakay) [reply virt/virt-4-12-release-notes.adoc:187]: @ksimon1 should we mention about known issues as well? I saw there is a section here or should we skip as it is a tech preview feature?

Comment (ksimon1) [reply virt/virt-4-12-release-notes.adoc:187]: Since this is a tech preview, I don't think it is necessary

**Comment by geetikakay** [comment]
lgtm

**Comment by sjhala-ccs** [comment]
@ksimon1 I have incorporated your suggestion. Please review again. Thank you.

**Pull request merged**